primitives/src/v0.rs

```diff
@@ -747,6 +748,18 @@ pub enum ValidityAttestation {
 }
 
 impl ValidityAttestation {
+	/// Produce the underlying signed payload of the attestation, given the hash of the candidate,
+	/// which should be known in context.
+	pub fn to_compact_statement(&self, candidate_hash: CandidateHash) -> CompactStatement {
+		// Explicit and implicit map directly from
+		// `ValidityVote::Valid` and `ValidityVote::Issued`, and hence there is a
+		// `1:1` relationshow which enables the conversion.
```

Comment:
Reviewers note: There is a guide section which states that there is a `1:1` mapping between `ValidityAttestation::Explicit <-> Vote` and `ValidityAttestation::Implicit <-> Seconded/Candidate` which was cross checked with the usage, so the above should be correct.

Reply:
Yep, that is correct.